packages/halo/src/react/index.js

```diff
@@ -29,43 +30,36 @@ const styles = {
       css[gapSize],
       css[gapTheme],
 
+      props.inline && css[`${BASE_CLASSNAME}--inline`],
       props.error && css[`${BASE_CLASSNAME}--error`],
       props.visible && css[visible],
       props.visibleOnFocus && css[visibleOnFocus]
     )
   }
 }
 
-const Halo = props => {
-  const {
-    error,
-    gapSize,
-    themeName,
-    shape,
-    visible,
-    visibleOnFocus,
-    ...filteredProps
-  } = props
-
-  return <div {...styles.halo(props)} {...filteredProps} />
-}
+const Halo = props => (
+  <div {...styles.halo(props)} {...filterReactProps(props)} />
```

Comment:
Is the pass-through of react props needed here?  I can't see the need yet.  What's your current thinking for where we're using this.  Also, since Halo isn't really a "user-facing" component, is there less of a need?

Reply:
it's not needed but i thought it looked cleaner so i used it.

i'm inclined to use the `filterReactProps` function often. I am running with the assumption it's safe to forward those react supported props. i could be persuaded to hold back if it's worrisome or we could make some guard rails to use